functional_tests/mp/06-share-child.yea

```diff
@@ -3,17 +3,13 @@ plugin:
   - wandb
 tag:
   shard: service
-  skip: true
 parametrize:
   permute:
     - :yea:start_method:
       - spawn
       - forkserver
```

Comment:
can you add fork?

Reply:
oops thought I merged it... haha let me push it